headers/validation.h

```diff
@@ -1264,14 +1264,15 @@ class Validator {
 			WA(name, ": Want integer, found nothing");
 		}
 		long long v;
-		auto begin = s.c_str(), end = begin + s.size();
-		auto [ptr, ec] = std::from_chars(begin, end, v);
-		if(ec == std::errc::result_out_of_range){
+		try {
+			auto begin = s.c_str(), end = begin + s.size();
+			auto [ptr, ec] = std::from_chars(begin, end, v);
+			if(ptr != end or ec != std::errc{})
+				WA(name, ": Parsing " + s + " as long long failed! Did not process all characters");
+		} catch(const std::out_of_range& e) {
 			WA(name, ": Number " + s + " does not fit in a long long!");
-		} else if(ptr != end) {
-			WA(name, ": Parsing " + s + " as long long failed! Did not process all characters");
-		} else if (ec != std::errc{}) {
-			WA(name, ": Parsing " + s + " as long long failed!");
+		} catch(const std::invalid_argument& e) {
+			WA("Parsing " + s + " as long long failed!");
```

Comment:
std::from_chars doesn't throw: https://en.cppreference.com/w/cpp/utility/from_chars

I think this hunk needs reverting

Reply:
Apparently, this `try`/`catch` block was there initially in the BAPC repo, but it was removed after that (in April: https://github.com/RagnarGrootKoerkamp/BAPCtools/commit/ad35772ef75974dd300aa8013b1efa5a8895b0f0). Thanks for checking! :smile: